### PR TITLE
Update Carbon Applications + Performance Tuning + Monitoring

### PR DESCRIPTION
**Pull request by YvonneW**
In addition to the file updates, the following files were removed:  
- view-and-download-logs.md
- performance-tuning.md
- monitoring-logs-using-management-console.md
- configuring-the-log-provider.md
- configuring-log4j-properties.md

### CODE REVIEW

File: en/docs/administer/capturing-system-data-in-error-situations.md [hunk 1]
```diff
@@ -40,33 +40,36 @@ inconsistencies.
...
+    ```shell tab="Windows"
+    carbondump.bat -carbonHome c:\wso2carbon-3.0.0\ -pid 5151
+    ```
 
-In Windows: carbondump.bat -carbonHome c:\wso2carbon-3.0.0\ -pid 5151
-```
 
 The tool captures the following information about the system:
 
--   Operating system information \*\* OS (kernel) version
```
Comment (gomathyk) [inline en/docs/administer/capturing-system-data-in-error-situations.md]: Are these forward slashes supposed to be there?

File: en/docs/administer/http-access-logging.md [hunk 2]
```diff
@@ -123,24 +86,14 @@ below to configure access logs to the HTTP servlet transport:
 3.  Restart the server. According to the default configurations, a log
     file named
     `           localhost_access_log_sample.{DATE}.log          ` is
-    created inside the \<
-    `           PRODUCT_HOME>/repository/logs          ` directory. The
+    created inside the `<I_HOME>/repository/logs          ` directory. The
```
Comment (gomathyk) [inline en/docs/administer/http-access-logging.md]: Please correct <IS_HOME>

File: en/docs/administer/jmx-based-monitoring.md [hunk 3]
```diff
@@ -121,139 +95,142 @@ directory).
...
-    -   [**MBeans**](#b94363057fb14c618354f2b55a7c9b43)
-
-    See the Oracle documentation on [using
-    JConsole](http://docs.oracle.com/javase/7/docs/technotes/guides/management/jconsole.html)
-    for more information on these tabs.
+4.  Click **Connect** to open the **Java Monitoring & Management
+    Console**. The following tabs will be available.
 
-    ![](../assets/img/53125400/57747081.png) 
+    - **Overview**: See the Oracle documentation on [using    JConsole](http://docs.oracle.com/javase/7/docs/technotes/guides/management/jconsole.html) for more information on these tabs.
```
Comment (gomathyk) [inline en/docs/administer/jmx-based-monitoring.md]: Please remove unnecessary space

File: en/docs/administer/capturing-system-data-in-error-situations.md (hunk 1 above)
Comment (YvonneW) [reply en/docs/administer/capturing-system-data-in-error-situations.md]: Fixed

**Pull request merged**